src/js/ripple/uint160.js

```diff
@@ -90,9 +92,9 @@ UInt160.prototype.to_json = function(opts) {
       }
 
       return output;
-    } else {
-      return this.to_hex();
     }
+    // else
```

Comment:
This is why I wasn't an advocate of this rule. Comments like this aren't good. We should either remove this comment or remove the eslint rule.